deploy/operator/common.sh

```diff
@@ -0,0 +1,35 @@
+__dir=${__dir:-"$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"}
+__root=${__root:-"$(realpath ${__dir}/../..)"}
```

Comment:
So the issues is other scripts are sourcing this module. For example, the deploy_spoke_cluster.sh which is in a different repository, and re-assigning the `__root` variable would mess the pipeline.
I would prefer those 2 variables only available on `common.sh` but unfortunately, `__dir` must be available in other scripts in order to source `common.sh`. 
So the solution I come up with is having them both in both places and just not overriding it in case they already exist.
Do you have another suggestion?